roofit/histfactory/src/HistFactoryModelUtils.cxx

```diff
@@ -243,6 +243,11 @@ namespace HistFactory{
     }
     delete obsIter;
 
+    if (!cat) {
+       std::cerr <<"Category not found"<< std::endl;
```

Comment:
This should rather be done with [RooFits own message service](https://root.cern/doc/master/rf506__msgservice_8C.html) I think.

But for now `std::cerr` is okay for me, I can follow up with another commit.

Reply:
Sorry, have no idea. I just hunting memory leaks - there are MUCH MUCH more